cypress/fixtures/customers/empty.json

```
@@ -0,0 +1,8 @@
+{
+    "customers": [],
+    "pageInfo": {
+      "currentPage": 1,
+      "totalPages": 1,
+      "totalCustomers": 0
+    }
+  }
```

```suggestion
  }

```

Recomenda-se deixar uma linha em branco ao final de cada arquivo para que ferramentas como o GitHub não exibam o ícone de `No newline at end of file`.

Isso é uma boa prática e convenção a ser seguida.